# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/tests/browser/browser.ini
# Commit: 812ed5275f0d
# Full Hash: 812ed5275f0db9f92944955f891df2ab68c9f8e7
# Author: Toshihito Kikuchi <tkikuchi@mozilla.com>
# Date: 2021-11-19 04:11:43
# Regressor Bug: 1734099
# File Overlap Count: 5
# Description:
#   Bug 1734099 - Never unload tabs with active RTCPeerConnection instances.  r=peterv,jesup
#   
#   This patch makes sure the Tab Unloading feature does not unload tabs that have
#   active peer connections not to disrupt browsing experience based on WebRTC peer
#   connections.
# ==============================================================================

diff -r a733d049dbda -r 812ed5275f0d dom/tests/browser/browser.ini
--- a/dom/tests/browser/browser.ini	Fri Nov 19 01:03:56 2021 +0200
+++ b/dom/tests/browser/browser.ini	Thu Nov 18 22:59:12 2021 +0000
@@ -63,6 +63,9 @@
 support-files =
   beforeunload_test_page.html
 run-if = e10s
+[browser_hasActivePeerConnections.js]
+support-files =
+  create_webrtc_peer_connection.html
 [browser_largeAllocation_win32.js]
 https_first_disabled = true
 skip-if = fission